### ISSUE TITLE
[Feature Request] Show totals on how many kanji have been covered

### ISSUE THREAD

**Issue by RogueArt** [Kuuuube/kanjigrid#9]
Hi, hope you're having a great day! I've been really enjoying using this extension, it's been incredibly helpful!

I've noticed a frequent need to calculate how many kanji I've covered and how many I have left to learn, which unfortunately the addon does not automatically do. For example, one of my common use cases is determining how many kanji I'm learning each week. Currently, I have to manually count these, though the sorting feature does make the process a bit faster.

I understand you're likely very busy, so I have two simple ideas for how this could be implemented:
![anki-kanji-grid](https://github.com/user-attachments/assets/ec3252f6-0945-4d58-b0a0-096292a88b61)

1. **Basic Progress Display:** - See next to the JLPT N5 section in the image, you could add a simple progress indicator like:  `(# of kanji covered at least once) / (# total kanji in that category)`. Basically, # of kanji that are not dark red divided by # all kanji in that category.
2. **Color-by-Color Breakdown:** See next to JLPT N2 section in the image - this is for users that want to know how immature or mature their cards are. For this, you could visualize it as a tuple/list, like `(# of red, # of orange, # of yellow, # of blue, total # of all kanji in category)`

At the very top, you could have aggregate totals across all JLPT levels. For example, while just the JLPT N5 subtotal would display (100/100), the overall total for all levels would either be shown as (1800/2186) or (30, 20, 150, 1600, 2186), depending on which implementation you go with.

Please let me know if this makes sense or if you need any further details or clarifications. Thank you so much for the great plugin, and I hope you have a wonderful day!

**Comment by Kuuuube** [comment]
Kanji grid does have a progress indicator. It's at the top of each category. Here's an example:

![image](https://github.com/user-attachments/assets/a1e0d2e2-ecde-4780-9be6-d02488f09a18)

The `31 of 80 Known` is what youre looking for.

Color by color breakdown seems like it might be more confusing than helpful.

**Comment by Kuuuube** [comment]
Closing as this appears stale and is unlikely to be implemented without further information.

**Comment by RogueArt** [comment]
Ahh I see the issue. This is the Kanji **Grid** addon, not the Kanji **Table** addon which I was using. I didn't realize they were two separate addons, since this is the top result when searching for the GitHub for "Kanji table"🤦‍♂️:

![Image](https://github.com/user-attachments/assets/aa3ec747-6034-4a4b-a37b-89066db34530)

I was very confused until now why I could never get the "Found" and "Known" count to appear on the addon despite being on the latest code, trying to reinstall it, and clearly seeing it in the repo's source code.

Anyways, thanks for closing this! I will find the right GitHub and send this suggestion there 😅.

I tried your addon and it has exactly the functionality I'm looking for with counting known and missing, so I'll be using it in the meantime. Thank you so much for your help!

**Comment by Kuuuube** [comment]
Hah, funny that I outdid kanji table's seo. But I think kanji grid is still by far the more popular one.

Also kanji table was created as a replacement for kanji grid which had been neglected for a long time and nobody wanted to maintain. Now that I've taken it up, there's been tons of bugs fixed and lots of features added. So you may want to give kanji grid a try and see how you feel about it.

**Comment by RogueArt** [comment]
> Now that I've taken it up, there's been tons of bugs fixed and lots of features added. So you may want to give kanji grid a try and see how you feel about it.

You're right, I remember last time, I couldn't even get the grid to render at all, so I had to find another plugin. But this time, I noticed this time that it worked right out of the box with no issues! 😄 Thanks for all the hard work on Kanji Grid, it's really nice to use now!!
